knjeru/fifi
# Add support for deleting debt item

**Changes:**
- Adds support for deleting a debt item
- Implements allowed origins for CORS
- Adds OPTIONS routes
- Fixes new debt functionality where the lack of a subscription was causing the request not to be submitted

**To Test:**
- Load app
- Go to the **Debts View**
- Submit a debt item
- Refresh to confirm the debt item was submitted
- Delete debt item
- Refresh to confirm the debt item was deleted
